### DISCUSSION TITLE
Cannot expand <iframe> height

### DISCUSSION THREAD

**Discussion by nil0x42** [TriliumNext/Trilium#6327] [Q&A]
I just created a render note, rendering an html code note based on iframe (to get original html display)

My html code note is like that:
```html
<iframe style="min-height: 85vh; width:100%; height:100%;" src="data:text/html;charset=utf-8,%3C!DOCTYPE......" sandbox=""></iframe>
```
The dirty `min-height: 85vh;` is the only way i found to display the iframe with an acceptable height.

If i use `min-height: 100%;`, the iframe's height is only 154px, and i could not find why...
Screenshot:
![Screenshot_2019-11-05_12-13-11](https://user-images.githubusercontent.com/3504393/68206960-dba25580-ffc5-11e9-9fb8-aad73fe2cbc1.png)

**Comment by HilkopterBob** [comment]
fix?!
